### PR TITLE
Updated dart sample

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: realm_dart/.gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+build/
+
+# Directory created by dartdoc
+doc/api/
+
+pubspec.lock
+*.realm
+*.realm.lock
+default.realm.management
+realm_dart_extension.dll
```
Comment (blagoev) [inline realm_dart/.gitignore:15]:
```suggestion
binary\windows\
binary\linux\
librealm_dart.dylib
```

File: realm_dart/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+## A simple command-line application using Realm Dart SDK
+
+## Setup
+
+* Dart SDK 2.12 stable is required from https://dart.dev/.
```
Comment (blagoev) [inline realm_dart/README.md:5]:
```suggestion
* Dart SDK ^2.15 stable is required from https://dart.dev/.
```

File: realm_dart/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    **Do not use the Dart SDK downloaded with Flutter 2.0 since it has issues and will not be able to run this sample**
```
Comment (blagoev) [inline realm_dart/README.md:7]: this file should be updated. take hints from realm-dart\README.md

File: realm_dart/bin/myapp.dart [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+part 'myapp.g.dart';
+
+@RealmModel()
+class _Car {
+  late String make;
+}
+
+@RealmModel()
+class _Person {
+  late String name;
```
Comment (blagoev) [inline realm_dart/bin/myapp.dart:30]: should we show object relationships?

File: realm_dart/build.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+targets:
```
Comment (blagoev) [inline realm_dart/build.yaml:1]: this file is not needed.

File: realm_dart/pubspec.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+
+dependencies:
+  realm_dart: 
+
+dev_dependencies:
+  pedantic: ^1.9.0
+  build_runner: ^2.1.2
```
Comment (blagoev) [inline realm_dart/pubspec.yaml:13]:
```suggestion

```

File: realm_dart/pubspec.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+publish_to: none
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+
+dependencies:
+  realm_dart: 
+
+dev_dependencies:
+  pedantic: ^1.9.0
```
Comment (blagoev) [inline realm_dart/pubspec.yaml:12]: remove pedantic

Comment (blagoev) [inline realm_dart/pubspec.yaml:12]:
```suggestion
  lints: ^1.0.1
```

**Review by blagoev — CHANGES_REQUESTED** [review]
great start. some changes suggested

File: realm_dart/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    This is needed only to regenerate Realm data model classes if they changed. 
+    If asked select `Delete` to delete any conflicts.
+
+    ```
+    dart run build_runner build
+    ``` 
+*  Run the application
+
+    ```
+    dart run myapp
```
Comment (blagoev) [inline realm_dart/README.md:43]:
```suggestion
    dart run
```

File: realm_dart/README.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+    ```
+    dart run realm_dart install
+    ```
+* [Optional] Generate Realm Realm data model classes
+    
+    This is needed only to regenerate Realm data model classes if they changed. 
+    If asked select `Delete` to delete any conflicts.
+
+    ```
+    dart run build_runner build
```
Comment (blagoev) [inline realm_dart/README.md:38]:
```suggestion
   dart run realm_dart generate
```

File: realm_dart/README.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+    ```
+* Install Realm Dart binary into the application
+
+    ```
+    dart run realm_dart install
+    ```
+* [Optional] Generate Realm Realm data model classes
+    
+    This is needed only to regenerate Realm data model classes if they changed. 
+    If asked select `Delete` to delete any conflicts.
```
Comment (blagoev) [inline realm_dart/README.md:35]:
```suggestion

```

File: realm_dart/README.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+    * On Mac
+
+    ```
+    export /Users/<YOUR_PATH>/dart-sdk.2.12.0/bin:$PATH
+    ```
+
+    * On Windows
+
+    ```
+    set PATH=C:\<YOUR_PATH>\dartsdk-windows-x64-release-2.12.0\bin;%PATH%
```
Comment (blagoev) [inline realm_dart/README.md:20]: these are not relevant now.

File: realm_dart/pubspec.yaml (hunk 7 above)
Comment (nielsenko) [reply realm_dart/pubspec.yaml:12]: Consider replacing with `lints: ^1.0.1`

File: realm_dart/pubspec.yaml [hunk 12]
```diff
@@ -0,0 +1,13 @@
+name: myapp
+description: A simple command-line application using Realm Dart SDK.
+publish_to: none
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+
+dependencies:
+  realm_dart:
```
Comment (nielsenko) [inline realm_dart/pubspec.yaml:9]:
```suggestion
  realm_dart: any
```
means the same, but state intent better

File: realm_dart/README.md (hunk 3 above)
Comment (desistefanova) [reply realm_dart/README.md]: Done.

File: realm_dart/build.yaml (hunk 5 above)
Comment (desistefanova) [reply realm_dart/build.yaml]: Deleted.

File: realm_dart/pubspec.yaml (hunk 7 above)
Comment (desistefanova) [reply realm_dart/pubspec.yaml]: Done.

File: realm_dart/README.md (hunk 11 above)
Comment (desistefanova) [reply realm_dart/README.md]: Removed.

**Review by blagoev — APPROVED** [review]

**Pull request merged**